default.nix

```diff
@@ -730,19 +617,19 @@ in let this = rec {
         iosReflexTodomvc
       ];
 
-
   demoVM = (import "${nixpkgs.path}/nixos" {
     configuration = {
       imports = [
         "${nixpkgs.path}/nixos/modules/virtualisation/virtualbox-image.nix"
         "${nixpkgs.path}/nixos/modules/profiles/demo.nix"
       ];
       environment.systemPackages = tryReflexPackages;
+      nixpkgs = { localSystem.system = "x86_64-linux"; };
```

Comment:
What does this do?

Reply:
This makes sure that we use linux to build the VM. This is only really relevant when you're on a macOS system where it will try to build NixOS but fail halfway through.

Reply:
Makes sense!